cgo/lapack_test.go

```diff
@@ -84,6 +84,18 @@ func (d blockedTranslate) Dorml2(side blas.Side, trans blas.Transpose, m, n, k i
 	impl.Dormlq(side, trans, m, n, k, a, lda, tau, c, ldc, work, len(work))
 }
 
+func (d blockedTranslate) Dorg2r(m, n, k int, a []float64, lda int, tau, work []float64) {
+	impl.Dorgqr(m, n, k, a, lda, tau, work)
+}
+
+func TestDorglq(t *testing.T) {
+	testlapack.Dorgl2Test(t, blockedTranslate{impl})
```

Comment:
On my machine, Dlantr doesn't panic. It does intermittently fail, but because of Dlange giving the wrong result.